src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java

```diff
@@ -280,6 +281,25 @@ private void processPosition(String path, Position p, boolean remove) {
 		updateMap.put(path + "." + Position.EVENT_SKATER, p.getSkater() == null ? null : p.getSkater().getId());
 		updateMap.put(path + "." + Position.EVENT_PENALTY_BOX, p.getPenaltyBox());
 	}
+	
+	private void processPenalties(Settings s) {
+		PenaltiesManager pm = new PenaltiesManager(s);
+
+        try {
+            PenaltiesDefinition penalties = pm.loadFromJSON();
+            for(PenaltyCode p : penalties.getPenalties()) {
```

Comment:
This needs to be broken down into multiple string entries, rather than one complex object that would require custom handling of some WS state entries by clients.